marcdomain/Fast-Food-Fast
# #160818761 Bg fix raw input validations

### What does this PR do?
- Implementation of Feedback on validation and naming conventions
### Description of Task to be completed
- Update validations to check for string inputs
- Update test scripts
- Update README
### How should this be manually tested?
- Use Postman to test all endpoints
- Check README for the test coverage
### Any background context you want to provide?
- String inputs on content-type application/json are validated here.
### What are the relevant pivotal tracker stories?
- [#160818761](https://www.pivotaltracker.com/story/show/160818761)
